CONTRIBUTING.md

```diff
@@ -36,6 +36,7 @@ brew uninstall --force foo
 brew install --build-from-source foo
 brew test foo
 brew audit --strict foo
+brew style foo
```

Comment:
Please make this a separate pull request.

Reply:
done